Sources/JExtractSwiftLib/SwiftTypes/SwiftKnownTypeDecls.swift

```diff
@@ -15,6 +15,7 @@
 import SwiftSyntax
 
 enum SwiftKnownTypeDeclKind: String, Hashable {
+  // MARK: -Std
```

Comment:
weird syntax? And we don't really say "std" about the stdlib -- how about just `// Swift` and `// Foundation` below